tests/test_measurement_model.py

```diff
@@ -0,0 +1,49 @@
+"""Testing the measurement model class."""
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+
+
+@pytest.fixture
+def create_mock_measurement() -> Measurement:
+    """Create a mock measurement instance."""
+
+    def _create_mock_measurement(
```

Comment:
Why is there a nested function, you can just return the Measurement directly. I'm actually surprised mypy doesn't error on this.

Reply:
Alternatively, if you need this as a function with variable inputs, you don't need the fixture. You can just write it as a function.

Reply:
I was just trying to get pytest.fixture to work... but yes this doesn't do anything actually and I can just use Measurement directly